src/components/DataViewFilterMultiselect/DataViewFilterMultiselectDropdownContent.tsx

```diff
@@ -14,119 +12,14 @@ import Button from "../Button";
 import ButtonRow from "../ButtonRow";
 import Spinner from "../Spinner";
 import CheckboxList from "@root/components/CheckboxList";
-import theme from "@root/theme";
+import Chip from "@root/components/Chip";
 import { H3 } from "../Typography";
 import { useMosaicTranslation } from "@root/i18n";
-
-const StyledWrapper = styled.div`
-	& > .topBlock {
-		display: flex;
-		height: 400px;
-		width: 600px;
-	}
-
-	& > .topBlock > .options > .searchBar {
-		position: sticky;
-		top: 0;
-		display: flex;
-		align-items: center;
-		padding-left: 14px;
-		z-index: 1;
-		background: white;
-	}
-
-	& > .topBlock > .options > .searchBar > svg {
-		margin-right: 8px;
-		font-size: ${theme.iconFontSize};
-	}
-
-	& > .topBlock > .options > .searchBar > .input {
-		font-family: ${theme.fontFamily};
-		line-height: 2rem;
-		font-size: 16px;
-	}
-
-	& > .topBlock > .options {
-		overflow-y: scroll;
-		flex: 2;
-		position: relative;
-		transition: opacity .5s;
-		padding-right: 10px;
-	}
-
-	& > .topBlock > .options.disabled {
-		opacity: .4;
-		pointer-events: none;
-	}
-
-	& > .topBlock > .options .listItem {
-		padding-top: 0px;
-		padding-bottom: 0px;
-	}
-
-	& > .topBlock > .options .listItemIcon {
-		min-width: auto;
-	}
-
-	& > .topBlock > .options > .spinner {
-		margin-top: 10px;
-		margin-bottom: 10px;
-	}
-
-	& > .topBlock > .options .checked {
-		color: ${theme.colors.blue};
-	}
-
-	& > .topBlock > .selected {
-		padding: 0px 0px 0px 15px;
-		width: 200px;
-		overflow-y: auto;
-	}
-
-	& .comparisonDropdown {
-		margin-bottom: 1rem;
-	}
-
-	& .chip {
-		display: flex;
-		background-color: ${theme.colors.blue}13;
-		margin-bottom: 4px;
-		justify-content: space-between;
-		font-family: ${theme.fontFamily};
-		height: 33px;
-	}
-
-	& .chip:focus {
-		background-color: ${theme.colors.blue}13;
-	}
-
-	& .chip > span {
-		flex: 1;
-		font-size: 14px;
-		text-overflow: ellipsis;
-		overflow: hidden;
-		white-space: nowrap;
-		display: block;
-	}
-`
-
-const PopoverP = styled.p`
-	margin: 0px;
-`
+import { ChipWrapper, PopoverP, StyledHr, StyledVerticalHr, StyledWrapper } from "./DataViewFilterMultiselect.styled";

```

Comment:
I can't add this in the line I want but please, update the whole div from line 277 with the following code:
`
<StyledTextField
						InputProps={{
							startAdornment: (
								<InputAdornment position="start">
									<SearchIcon/>
								</InputAdornment>
							),
						}}
						className="searchBar"
						placeholder={props.placeholder || t("mosaic:common.keyword___")}
						autoFocus={true}
						onChange={keywordChange}
						fieldSize="100%"
					/>
`

This will allow us to have the same style amongst all TextFields.